server/lib/sendMessagesToAdmins.js

```diff
@@ -0,0 +1,46 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Roles, Users } from '../../app/models/server';
+
+export function sendMessagesToAdmins({
+	fromId = 'rocket.cat',
+	checkFrom = true,
+	msgs = [],
+	banners = [],
+}) {
+	const fromUser = checkFrom ? Users.findOneById(fromId, { fields: { _id: 1 } }) : true;
+
+	Roles.findUsersInRole('admin').forEach((adminUser) => {
+		if (fromUser) {
```

Comment:
does this mean?
```
if (!fromUser) {
  return;
}
```

if so, if can be done outside the `forEach` loop as well

Reply:
The banner insertion doesn't depend on fromUser